spec/datadog/ci/contrib/rspec/instrumentation_spec.rb

```diff
@@ -967,7 +967,7 @@ def rspec_skipped_session_run
       let(:integration_options) { {service_name: "lspec"} }
     end
 
-    it "retries test until it passes" do
+    it "does not report the test that failed when RSpec was quitting" do
```

Comment:
This change is not relevant to this PR's topic, I just noticed that one of the specs was copy-pasted without changing description